v20.1/time.md

```diff
@@ -1,39 +1,68 @@
+{{site.data.alerts.end}}
+
 ## Aliases
 
 In CockroachDB, the following are aliases:
 
-`TIME WITHOUT TIME ZONE`
+Alias    | Long Version
+---------|-------------
+`TIME`   | `TIME WITHOUT TIME ZONE`
+`TIMETZ` | `TIME WITH TIME ZONE`
 
 ## Syntax
 
-A constant value of type `TIME` can be expressed using an
-[interpreted literal](sql-constants.html#interpreted-literals), or a
-string literal
-[annotated with](scalar-expressions.html#explicitly-typed-expressions)
-type `TIME` or
-[coerced to](scalar-expressions.html#explicit-type-coercions) type
-`TIME`.
+### `TIME`
 
-The string format for time is `HH:MM:SS.SSSSSS`. For example: `TIME '05:40:00.000001'`.
+A constant value of type `TIME` can be expressed using an [interpreted literal](sql-constants.html#interpreted-literals), or a string literal [annotated with](scalar-expressions.html#explicitly-typed-expressions) type `TIME` or  [coerced to](scalar-expressions.html#explicit-type-coercions) type `TIME`. When it is unambiguous, a simple unannotated [string literal](sql-constants.html#string-literals) can also be automatically interpreted as type `TIME`.
 
-When it is unambiguous, a simple unannotated [string literal](sql-constants.html#string-literals) can also
-be automatically interpreted as type `TIME`.
+The string format for `TIME` is `HH:MM:SS.SSSSSS`. For example: `TIME '05:40:00.000001'`. The fractional portion is optional and is rounded to microseconds (i.e., six digits after the decimal) for compatibility with the [PostgreSQL wire protocol](https://www.postgresql.org/docs/current/static/protocol.html).
 
-Note that the fractional portion of `TIME` is optional and is rounded to microseconds (i.e., six digits after the decimal) for compatibility with the [PostgreSQL wire protocol](https://www.postgresql.org/docs/current/static/protocol.html).
+{{site.data.alerts.callout_info}}
+A date of `0000-01-01` is displayed for all `TIME`/`TIMETZ` values, but is not stored in the database. To print without a date, you can cast the type to a `STRING`.
+
+A time zone offset of `+00:00` is also displayed for all `TIME` and [`TIMESTAMP`](timestamp.html) values, but is not stored in the database.
+{{site.data.alerts.end}}
+
+### `TIMETZ`
+
+To express a `TIMETZ` value with a time zone offset from UTC, you can add an offset to a `TIME` value. For example, `TIMETZ '10:10:10.555555-05:00'` offsets from UTC by -5.
+
+If no time zone is specified for a `TIMETZ` value, the `timezone` [session variable](show-vars.html#supported-variables) is used. For example, if you [set the `timezone`](set-vars.html#set-time-zone) for a session using `SET TIME ZONE -2`, and you define the `TIMETZ` as `TIMETZ '10:10:10.55'`, the value will be displayed with an offset of -2 from UTC.
+
+`TIMETZ` is not affected by session-scoped offsets (unlike [`TIMESTAMPTZ`](timestamp.html)). Time zone offsets only apply to values inserted after the offset has been set, and do not affect existing `TIMETZ` values.
```

Comment:
minor nit: `apply to values inserted after the offset has been set` can be a bit misleading, e.g. `SET TIME ZONE +3` then `SELECT '12:00-03:00'::timetz` will use `-3` as a priority over `+3`.